include/jemalloc/internal/quantum.h

```diff
@@ -24,7 +24,7 @@
 #  ifdef __arm__
 #    define LG_QUANTUM		3
 #  endif
-#  ifdef __aarch64__
+#  if defined(__aarch64__) || defined(_M_ARM64)
```

Comment:
Why do we need to add `_M_ARM64` here?

Reply:
__aarch64__ is a predefined macro by clang/gcc, and _M_ARM64 is a predefined macro by MSVC/clang-cl. This change should equalize LG_QUANTUM defined for all mainstream compilers, IIRC. This is similar to line 21, or below. __x86_64__ is a macro predefined by clang/gcc, while _M_X64 is a predefined macro by MSVC/clang-cl.
`#  if (defined(__amd64__) || defined(__x86_64__) || defined(_M_X64))`

Reply:
There seem only four places that check __aarch64__, see below. The first two don't need a fix because SYS_open and __APPLE__ are not defined on Windows. The one in quantum.h is already handled by this PR. Only the last one needs some investigation. If you could open another PR to address it, that'll be great. Thanks.
```
E:\jemalloc>findstr /Ls __aarch64__ *.h
include\jemalloc\internal\jemalloc_internal_decls.h:#    if defined(SYS_open) && defined(__aarch64__)
include\jemalloc\internal\jemalloc_internal_inlines_a.h:#elif defined(__aarch64__) && defined(__APPLE__)
include\jemalloc\internal\quantum.h:#  if defined(__aarch64__) || defined(_M_ARM64)
include\jemalloc\internal\rtree.h:#    ifdef __aarch64__
```